lib/templates/codecov.yml

```
@@ -0,0 +1,17 @@
+coverage:
+  precision: 2
```

@ElseThen can we make it `precision: 1` ? `2` was quite noisy in some projects.

```suggestion
  precision: 1
```